### PR TITLE
When indexing ContentAggregators (which index fulltext of children in…

### PR DESCRIPTION
**Pull request by elohanlon**
…to parent fulltext field), use risearch query as more efficient way of handling many children; Also enable focus param on tests

### CODE REVIEW

File: lib/cul_hydra/risearch_members.rb [hunk 1]
```diff
@@ -59,6 +59,39 @@ def get_direct_member_count(pid, flush_resource_index_before_query=false, verbos
...
+    #   "select $pid from <#ri>
+    #   where $pid <http://purl.oclc.org/NET/CUL/memberOf> <fedora:#{pid}>
+    #   and $pid <fedora-view:disseminates> $ds
+    #   and $ds <fedora-view:disseminationType> <info:fedora/*/#{dsid}>"
+
+    direct_members_with_ds_query = "select $pid
+      count( select $ds from <#ri> where  $pid <fedora-view:disseminates> $ds and $ds <fedora-view:disseminationType> <info:fedora/*/#{dsid}>)
+      from <#ri>
+      where $pid <http://purl.oclc.org/NET/CUL/memberOf> <info:fedora/#{pid}>
+      and $pid <fedora-view:disseminates> $ds
```
Comment (barmintor) [inline lib/cul_hydra/risearch_members.rb]: remove the disseminates clause outside the count subquery?

Comment (elohanlon) [reply lib/cul_hydra/risearch_members.rb]: Will do, thanks!

**Comment by elohanlon** [comment]
Travis CI is having some problems at the moment, but I can confirm that CI tests pass when I run them on my local machine.

**Comment by barmintor** [comment]
👍

**Pull request merged**